cmd/medius/common/registry.go

```diff
@@ -124,7 +133,10 @@ func NewRegistry() []Entry {
 	registry := make([]Entry, len(staticRegistry))
 	copy(registry, staticRegistry)
 
-	gatherers := []api.ArtifactsGatherer{fedora.NewGatherer()}
+	gatherers := []api.ArtifactsGatherer{
+		fedora.NewGatherer("39", []string{"x86_64", "aarch64"}),
```

Comment:
That is not ok, the gatherer needs to be able to dynamically retrieve versions. We cannot make it fixed to certain versions because it defeats the whole purpose of the gatherer.